modules/common/src/main/scala/notebook/front/widgets/DataFrame.scala

```diff
@@ -0,0 +1,117 @@
+package notebook.front.widgets
```

Comment:
this file should be put int the `post-df` folder otherwise package pre spark 1.3 won't compile -_-... I know, I know